system/jlib/jsocket.cpp

```diff
@@ -964,7 +964,7 @@ void CSocket::open(int listen_queue_size,bool reuseports)
 #ifndef _WIN32
     reuseports = true;  // for some reason linux requires reuse ports
 #endif
-    if (reuseports) {
+    if (reuseports && listen_queue_size) {
```

Comment:
this may make sense, but not sure how it's related to the change?

Reply:
I wanted to make sure we bound to an unused port.
TBH unless the port chosen is 'fixed' (ir dali, thor etc.) we should not set SO_REUSEADDR.

Reply:
that makes sense.
Can you add a comment at head of this function, describing the use case when listen_queue_size == 0

Reply:
Added.